clients/fides-js/src/integrations/blueconic.ts

```diff
@@ -0,0 +1,55 @@
+declare global {
+  interface Window {
+    blueConicClient?: {
+      profile: {
+        setConsentedObjectives: (objectives: string[]) => void;
+        setRefusedObjectives: (objectives: string[]) => void;
+        updateProfile: () => void;
+      };
+      event: {
+        subscribe: any;
+      };
+    };
+  }
+}
+
+// https://support.blueconic.com/hc/en-us/articles/202605221-JavaScript-front-end-API
+const blueConicLoaded = () =>
+  typeof window.blueConicClient !== "undefined" &&
+  typeof window.blueConicClient.event !== "undefined" &&
+  typeof window.blueConicClient.event.subscribe !== "undefined";
+
+const configureObjectives = () => {
+  if (!blueConicLoaded() || !window.blueConicClient?.profile) {
+    return;
+  }
+
+  const profile = window.blueConicClient?.profile;
+  const { consent } = window.Fides;
```

Comment:
This feels like a potential race condition, assuming this `window.Fides` will already be updated after the event. Feels safer to get the consent object from the event itself if possible.

```
  configureObjectives = (event?: FidesEvent) => {
```

and then

```suggestion
  const { consent } = event?.details || window.Fides;
```

Reply:
I'm down to change it that way but... our docs are _very_ explicit that this would not be a race condition:

> **FidesUpdated:** Dispatched when a user action (e.g. accepting all, saving changes, applying GPC) has finished updating the user's consent preferences. **This event is dispatched once the changes are fully saved to the Fides object, fides_consent cookie on the user's device, and the Fides API.** To receive an event that fires before these changes are saved, use the FidesUpdating event instead.

> **FidesInitialized**: Dispatched when initialization is complete and the current user's consent preferences - either previously saved or applicable defaults - **have been set on the Fides global object.**

Emphasis mine.

Reply:
Ah perfect. Thanks for the sanity check. I can't think of any reason not to do it this way in that case. It's definitely cleaner code.